docs/specifications/tests/DNSSEC-TP/dnssec13.md

```diff
@@ -152,20 +157,29 @@ See the [DNSSEC README] document about DNSSEC algorithms.
 
 Test case is only performed if DNSKEY records are found.
 
+It is assumed that [DNSSEC08] and [DNSSEC09] have reported on any
+unexpected errors on retreiving the DNSKEY records and SOA record,
+respectively, from *Child Zone*.
+
+It is assumed that there are no unexpected errors on retreiving the
+NS records from *Child Zone* that have not been reported by
+[Basic04], [DNSSEC08] and [DNSSEC09].
```

Comment:
I'm thinking of the location of these statements. Ideally they should be located in a "Scope" section, but it's seems we don't have those - I thought we did. In the absence of such a section, I think "Objective" or "Intercase dependencies" would be better locations than "Special procedural requirements". What are your thoughts on this?